kernel-spark/src/test/java/io/delta/kernel/spark/snapshot/CatalogManagedSnapshotManagerTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (2025) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.spark.snapshot;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.apache.hadoop.conf.Configuration;
+import org.junit.jupiter.api.Test;
+
+/** Tests for {@link CatalogManagedSnapshotManager}. */
+class CatalogManagedSnapshotManagerTest {
```

Comment:
Can you try using the sample catalog managed table used by `kernelUnityCatalog`?